.travis.yml

```diff
@@ -3,11 +3,23 @@ services:
   - docker
 
 script:
+  # Shellcheck
   - shellcheck *.sh
-  - shellcheck ci-deploy/*.sh
-  - cd .. &&
-    git clone https://github.com/whatwg/html.git --depth 2 &&
-    IS_TEST_OF_HTML_BUILD_ITSELF=true bash ./html-build/ci-deploy/outside-container.sh
```

Comment:
It's a clear name, but I don't think I'll miss `IS_TEST_OF_HTML_BUILD_ITSELF` much :)